command/data_format_test.go

```diff
@@ -1,10 +1,10 @@
 package command
 
 import (
-	"strings"
 	"testing"
 
 	"github.com/hashicorp/nomad/ci"
+	"github.com/shoenig/test/must"
 )
 
 type testData struct {
```

Comment:
I think all of these can be declared inside `TestDataFormat`? They seem specific to that test.